src/cmd/package.go

```diff
@@ -37,7 +37,7 @@ var packageCmd = &cobra.Command{
 var packageCreateCmd = &cobra.Command{
 	Use:     "create [ DIRECTORY ]",
 	Aliases: []string{"c"},
-	Args:    cobra.MaximumNArgs(1),
+	Args:    cobra.ExactArgs(1),
```

Comment:
```suggestion
	Args:    cobra.MaximumNArgs(1),
```


`zarf package create` is a thing (with no args)